junit-platform-launcher/src/main/java/org/junit/platform/launcher/TestIdentifier.java

```diff
@@ -50,10 +42,6 @@ public final class TestIdentifier implements Serializable {
 
 	@Serial
 	private static final long serialVersionUID = 1L;
-	@Serial
-	@SuppressWarnings("UnusedVariable")
-	private static final ObjectStreamField[] serialPersistentFields = ObjectStreamClass.lookup(
-		SerializedForm.class).getFields();
 
 	// These are effectively final but not technically due to late initialization when deserializing
 	private /* final */ UniqueId uniqueId;
```

Comment:
This is no longer the case. Please remove this comment and convert `/* final */` to `final` on all fields.

```suggestion
	private final UniqueId uniqueId;
```